node-runner-cli/api/ValidatorConfig.py

```diff
@@ -108,12 +108,30 @@ def add_change_ownerid(actions: List, validator_info: EntityResponse):
         owner_id = input("\nEnter the new owner id or press Enter not to change:").strip()
         if owner_id != "":
             if owner_id != current_value:
-                actions.append(Action().set_validator_owner(owner_id))
+                actions.append(Action.set_validator_owner(owner_id))
                 return actions
             Helpers.print_coloured_line("Owner entered is same . So action will not be applied", bcolors.WARNING)
 
         return actions
 
+    @staticmethod
+    def vote(actions: List, validator_info: EntityResponse, health, engine_configuration):
+        print("-------- Vote -----\n")
+        latest_fork_name = print_vote_and_fork_info(health, engine_configuration)        
+        response = input(f"{bcolors.BOLD}\nDo you want to cast a vote for fork '{latest_fork_name}' [y/n]? " +
+                         f"Presssing Enter does not cast a vote: {bcolors.ENDC}").strip()
```

Comment:
Pressing*. Also, what do you mean pressing enter does not cast a vote? Do you mean that entering "" counts as "n"?